docs/1.16/prisma-client/api-reference/check-existence-GO-pyl3.mdx

```diff
@@ -0,0 +1,24 @@
+import Collapse from 'components/Markdown/Collapse'
+
+// TODO: This Go API is incomplete, see TS
```

Comment:
unfortunately `//`-comments are rendered as normal text, currently I don't know of a way to do comments in MDX. regular HTML comments also don't work.